canonical-web-and-design/jaas-dashboard
# Add table empty state to models tables

## Done

Add table empty state to models tables

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Observe that the empty state appears

## Details

Fixes #122
